aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -347,6 +347,14 @@ static void trigamma_kernel(TensorIterator& iter) {
   });
 }
 
+static void exp2_kernel(TensorIterator& iter) {
```

Comment:
Can this be analogous to the implementation of exp, here:

https://github.com/pytorch/pytorch/blob/89ac30afb88451efc66ed8077b7bbb222afd3428/aten/src/ATen/native/cpu/UnaryOpsKernel.cpp#L642

?

Reply:
Although I suppose you'd be doing IMPLEMENT_FLOAT_KERNEL, instead.

Reply:
https://github.com/pytorch/pytorch/blob/f9a0d0c21e0fd66f003931019c4dcc643b52e291/aten/src/ATen/native/cpu/UnaryOpsKernel.cpp#L531-L560

`IMPLEMENT_FLOAT_KERNEL` expects vectorised implementation. So we with the current implementation.

Reply:
Oh I see. A comment explaining why exp2 is implemented differently than exp would be helpful.